module/mempool/stdmap/incorporated_result_seals.go

```diff
@@ -46,15 +154,73 @@ func (ir *IncorporatedResultSeals) ByID(id flow.Identifier) (*flow.IncorporatedR
 
 // Rem removes an IncorporatedResultSeal from the mempool
 func (ir *IncorporatedResultSeals) Rem(id flow.Identifier) bool {
-	return ir.Backend.Rem(id)
+	removed := false
+	err := ir.Backend.Run(func(entities map[flow.Identifier]flow.Entity) error {
+		entity, ok := entities[id]
+		if ok {
+			seal := entity.(*flow.IncorporatedResultSeal)
+			removeSeal(seal, entities, ir.byHeight)
+			removed = true
+		}
+		return nil
+	})
+	if err != nil {
+		panic(err)
+	}
+	return removed
 }
 
-// Clear removes all entities from the pool.
 func (ir *IncorporatedResultSeals) Clear() {
-	ir.Backend.Clear()
+	err := ir.Backend.Run(func(entities map[flow.Identifier]flow.Entity) error {
+		for k := range entities {
+			delete(entities, k)
+		}
```

Comment:
This is logic that `Backdata` already implements (and more efficiently). No need to re-implement.